File: src/exercises/bowling/ocaml/template.ml
type game = { rolls : int list }

let create () : game = { rolls = [] }

let roll (pins : int) (g : game) : (game, string) result =
  (* Your code here *)
  Ok g

let score (g : game) : (int, string) result =
  (* Your code here *)
  Ok 0

File: src/exercises/bowling/go/template.go
package main

type Game struct{}

func NewBowling() *Game {
	return &Game{}
}

func (g *Game) Roll(pins int) error {
	// Your code here
	return nil
}

func (g *Game) Score() (int, error) {
	// Your code here
	return 0, nil
}
